user.css

```diff
@@ -57,3 +57,12 @@
 .desktoproutes-homepage-takeover-ad-hptoComponent-parentContainer{
   display: none !important;
 }
+
+/*Remove download buttons*/
+.x-downloadButton-DownloadButton{
+  display:none;
+}
+
+.main-contextMenu-menuItem:nth-of-type(8) {
```

Comment:
Unfortunately, grabbing elements like this by the list number won't work.
For example, for non-user-created libraries, the download button is not the 8th item on the list and it won't remove the download button.
![image](https://user-images.githubusercontent.com/43648146/127741942-0e193969-e980-4c70-9738-a2f096ad570d.png)

I'll see if I can do anything to fix this issue.